### PR TITLE
The implementation of basic functionality that reflects the principle of polymorphism.

### PR DESCRIPTION
**Pull request by malikalhack**
The list of changes:
- Added a new project to the solution
- Added new files for the circle class
- Fixed typos
- Fixed bugs
- Added missing Doxygen notes
- Updated README.md file

### CODE REVIEW

File: Inheritance/inc/rectangle.h [hunk 1]
```diff
@@ -3,21 +3,23 @@
...
  * @authors Anton Chernov
  * @date    29/10/2022
- * @date    30/10/2022
+ * @date    02/11/2022
  */
 
 #ifndef RECTANGLE_H
 #define RECTANGLE_H
-/****************************** Included files ********************************/
+ /****************************** Included files ********************************/
```
Comment (malikalhack) [inline Inheritance/inc/rectangle.h]: Remove extra space

File: Inheritance/inc/rectangle.h [hunk 2]
```diff
@@ -3,21 +3,23 @@
...
 #include "coordinate.h" /* coordinate class interface */
 /******************************** Definition **********************************/
 #define WIDTH_LIMIT     (50u)
 #define LENGTH_LIMIT    (50u)
 
-#if !(WIDTH_LIMIT && LENGTH_LIMIT)
-    #error Rectangle limits are wrong
+#if (LENGTH_LIMIT < X_LIMIT - 2) && (WIDTH_LIMIT < Y_LIMIT - 2)
+    #if !(WIDTH_LIMIT && LENGTH_LIMIT)
```
Comment (malikalhack) [inline Inheritance/inc/rectangle.h]: Something is wrong with the condition.

File: Polymorphism/inc/circle.h [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+#define CIRCLE_H
+/****************************** Included files ********************************/
+#include "coordinate.h" /* coordinate class interface */
+/******************************** Definition **********************************/
+#define PI 3.14159265359f
+
+#define RADIUS_LIMIT     (50u)
+
+#if (RADIUS_LIMIT < X_LIMIT - 2) && (RADIUS_LIMIT < Y_LIMIT - 2)
+#if !(RADIUS_LIMIT)
```
Comment (malikalhack) [inline Polymorphism/inc/circle.h]: The condition is wrong

File: Common/src/common.c [hunk 4]
```diff
@@ -0,0 +1,14 @@
+/**
+ * @file    common.h
```
Comment (malikalhack2) [inline Common/src/common.c]: Fix name of file

**Review by malikalhack2 — APPROVED** [review]

**Pull request merged**